gateway/app/main/processes/controller.py

```diff
@@ -1,11 +1,83 @@
 from .federated_learning_process import FLProcess
+from .federated_learning_cycle import FederatedLearningCycle
 
 
 class FLPController:
```

Comment:
This class now controls cycles and processes. Do you think it would be a good idea to rename it to `FLController`?

Reply:
Agreed @IonesioJunior

Reply:
Yep, I agree with it! :)

Reply:
Nice :+1: